doc/tutorial/get_started.md

```diff
@@ -138,7 +140,7 @@ Complete the following steps to set up this network:
 
        lxc network create microbr0
 
-(tutorial-note-ips)=
+      (tutorial-note-ips)=
```

Comment:
this was recently introduced by @minaelee , not sure what's about, but could you check if it still behaves as expected while indented? otherwise it breaks the number listing of the section